packages/ui5-proxy-middleware/src/base/ui5.d.ts

```diff
@@ -0,0 +1,118 @@
+declare module '@ui5/fs' {
```

Comment:
I understand why we need to have our own types declaration file. @tobiasqueck, do you know if the UI5 libraries have plans to supply the types in the future? Does it make sense to contribute to their repos?

Reply:
Already requested at https://github.com/SAP/ui5-tooling/issues/651.